python/cudf/cudf/_lib/orc.pyx

```diff
@@ -163,6 +163,8 @@ cdef compression_type _get_comp_type(object compression):
         return compression_type.ZLIB
     elif compression == "ZSTD":
         return compression_type.ZSTD
+    elif compression == "lz4":
```

Comment:
I'm trying to find a reference on the supported values here. The tests are inconsistent about using `compression="lz4"` vs. `compression="LZ4"`.

It looks like on one page of its docs, the PyArrow ORC reader uses lowercase values for `compression`, but we have uppercase `ZLIB` and `ZSTD` here. `lz4` isn't listed. On another page, the values are all uppercase. Can we double-check this and make our casing match the right part of the PyArrow API spec?

https://arrow.apache.org/docs/python/orc.html#compression
https://arrow.apache.org/docs/python/generated/pyarrow.orc.ORCWriter.html

Reply:
We need to document/support both lower-case and upper-case strings for `compression` here. Pandas just uses Pyarrow in this case, and pyarrow supports both upper and lower case strings: https://github.com/apache/arrow/blob/d9891918a42c74002533ed5c06c42b7d0d070820/python/pyarrow/_orc.pyx#L56 So we just need to do `compression.upper()` too

Reply:
hmm, upper() would also allow `Lz4` and `lZ4`. 
Should we specifically allow `LZ4` and `lz4`, or just make the option case-insensitive? Thoughts, @bdice ?

Reply:
I just realized what the link above points to. `upper()` it is!